config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Bundle.java

```diff
@@ -202,4 +202,34 @@ public void setFolderTree(FolderTree folderTree) {
     public void setDependencyMap(Map<Dependency, List<Dependency>> dependencyMap) {
         this.dependencyMap = dependencyMap;
     }
+
+    public void verifyExistingFolderPath(String folderPath, Bundle bundle) {
+        Boolean isValidImport = false;
+
+        // True because there isn't a specific directory. Only requires Root Node to exist.
+        if (folderPath.equals("/")) {
+            isValidImport = true;
+        }
+        
+        // Compares the folder tree path structure with specified Gateway connection folder path
+        String[] folderList = folderPath.split("/");
+        if (folderList.length > 0) {
+            String leafFolderName = folderList[folderList.length - 1];
+            Folder leafFolder = new Folder();
+            for (Folder folder : bundle.getFolders().values()) {
+                if (folder.getName().equals(leafFolderName)) {
+                    leafFolder = folder;
+                }
+            }
+
+            String leafFolderPath = "/" + leafFolder.getPath();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this hard-coded path-delimiter. ([squid:S1075](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1075))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=131&issues=AWcZobkXkEeD8QRklwaY&open=AWcZobkXkEeD8QRklwaY)

[ISSUE_KEY]: AWcZobkXkEeD8QRklwaY